doc/API.md

```diff
@@ -219,7 +219,7 @@ This objects represents the server connection. You can create new ```JitsiConnec
     - appID - identification for the provider of Jitsi Meet video conferencing services. **NOTE: not implemented yet. You can safely pass ```null```**
     - token - secret generated by the provider of Jitsi Meet video conferencing services. The token will be send to the provider from the Jitsi Meet server deployment for authorization of the current client.
     - options - JS object with configuration options for the server connection. You can change the following properties there:
-        1. bosh -
+        1. serviceUrl - XMPP service URL. For  example 'wss://server.com/xmpp-websocket' for Websocket or '//server.com/http-bind' for BOSH.
```

Comment:
Maybe document the deprecated one (saying it’s deprecated) to avoid confusion?

Reply:
done!